engine/src/main/java/org/terasology/logic/console/commands/CoreCommands.java

```diff
@@ -506,24 +526,91 @@ public String spawnBlock(@Sender EntityRef sender, @CommandParam("blockName") St
         spawnPos.add(offset);
 
         BlockFamily block = blockManager.getBlockFamily(blockName);
-        if (block == null) {
-            return "";
-        }
 
         BlockItemFactory blockItemFactory = new BlockItemFactory(entityManager);
         EntityRef blockItem = blockItemFactory.newInstance(block);
 
         blockItem.send(new DropItemEvent(spawnPos));
         return "Spawned block.";
     }
+    @Command(shortDescription = "Drops many blocks as per value choosen by the player", helpText = "the particular block can be seleted which the player wants to drop"
+                , runOnServer = true, requiredPermission = PermissionManager.CHEAT_PERMISSION)
+    public String bulkDropper(@Sender EntityRef sender, @CommandParam("blockName") String blockName, @CommandParam("value") int value) {
+
+        //This is a loop which gives the particular amount of block the player wants to spawn
+        ClientComponent clientComponent = sender.getComponent(ClientComponent.class);
+        LocationComponent characterLocation = clientComponent.character.getComponent(LocationComponent.class);
+
+        Vector3f spawnPos = characterLocation.getWorldPosition();
+        Vector3f offset = characterLocation.getWorldDirection();
+        offset.scale(3);
+        spawnPos.add(5,10,0);
+        BlockFamily block = blockManager.getBlockFamily(blockName);
+        if (block == null) {
+            return "";
+        }
+        BlockItemFactory blockItemFactory = new BlockItemFactory(entityManager);
+
+        for (int i = 0; i <= value; i++) {
+
+            EntityRef blockItem = blockItemFactory.newInstance(block);
+
+            blockItem.send(new DropItemEvent(spawnPos));
+        }
+
+        // this returns the block you have spawned and the amount
+        return "Dropped " + value + " " + blockName + " Blocks :)";
+    }
 
+    @Command(shortDescription = "Arranges the blocks in a bowling pin format.", helpText = "Spawns a specific blocks in a regular bowling pin pattern, The item " +
+            " front of the player can simply picked up",
+            runOnServer = true, requiredPermission = PermissionManager.CHEAT_PERMISSION)
+    public String prepare  (@Sender EntityRef sender, @CommandParam("blockName") String blockName) {
```

Comment:
Requesting a command rename here: `prepare` is very generic and suggests nothing about what is being prepared. Maybe instead "bowlingPrep` or `bowlingPins` or something in that vein. Look for a command that's unique enough so auto-complete will have no trouble with ambiguous leading letters. 

Text could also be improved a little bit ("Arranges the blocks" is bad grammar - which blocks? Could as well be "Sets up a typical bowling pin arrangement in front of the player" or so)